vscode-lean4/src/infoview.ts

```diff
@@ -606,7 +606,7 @@ export class InfoProvider implements Disposable {
     private async handleInsertText(text: string, kind: TextInsertKind, uri?: Uri, pos?: Position) {
         let editor: TextEditor | undefined
         if (uri) {
-           editor = window.visibleTextEditors.find(e => e.document.uri === uri);
+           editor = window.visibleTextEditors.find(e => e.document.uri.path === uri.path);
```

Comment:
```suggestion
           editor = window.visibleTextEditors.find(e => e.document.uri.toString() === uri.toString());
```

Before somebody complains that we're inserting text in `http://example.com/home/gebner/foo.lean` instead of `/home/gebner/foo.lean`. :smile: